### ISSUE TITLE
Read & Write AspNet 4.X Authentication Ticket Cookies form .Net Core

### ISSUE THREAD

**Issue by akrock** [dmarlow/AspNetTicketBridge#4]
We had a legacy auth system that was using the old Authentication Tickets, and wanted to have our new Authentication System in .NET Core interop with it.

This library already contained much of the pieces, but needed a little bit more boilerplate to make it easy to do.

Here is an example of this working:
```
private static void SetCookieOptions(CookieAuthenticationOptions opts, B2CSettings settings)
        {
            opts.Cookie.Domain = settings.CookieDomain;
            opts.Cookie.Name = "CookieName";
            opts.Cookie.Expiration = new TimeSpan(4, 30, 0);
            opts.Cookie.SecurePolicy = CookieSecurePolicy.Always;
            opts.Cookie.HttpOnly = true;
            opts.Cookie.IsEssential = true;
            opts.ExpireTimeSpan = new TimeSpan(4, 30, 0);
            opts.Cookie.Path = "/";

            //Interop support for cookies from ASP.NET 4.x
            opts.TicketDataFormat =
                new AspNet4TicketDataFormat(new MachineKeyDataProtector(
                        settings.ValidationKey, settings.DecryptionKey)
                    .ForPurposes(MachineKeyDataProtector.DefaultCookiePurposes));
        }
```

Will clean-up what I added an submit a PR shortly.

**Comment by dmarlow** [comment]
Looks great. I've merged it in and published an updated package to NuGet (v1.4.0). Thanks for helping!

**Comment by akrock** [comment]
Thank you for putting this out there, and being so quick to respond!

🥇 support for your library 😃
